apps/vercel/src/locations/ConfigScreen.tsx

```diff
@@ -98,6 +101,21 @@ const ConfigScreen = () => {
     getContentTypes();
   }, []);
 
+  useEffect(() => {
```

Comment:
I lifted this (and `contentTypes` and we can talk about following up with injecting these lists if that makes sense @jsdalton